airflow/config_templates/config.yml

```diff
@@ -1750,7 +1750,14 @@
       default: "IfNotPresent"
     - name: delete_worker_pods
       description: |
-        If True (default), worker pods will be deleted upon termination
+        If True, all worker pods will be deleted upon termination
+      version_added: ~
+      type: string
+      example: ~
+      default: "True"
+    - name: delete_worker_pods_on_success
```

Comment:
Has this already landed upstream?

If not, then I would change this around -  `delete_worker_pods_on_failure`

Reply:
The reason I say this is looking at the config setting, I went "of course I want to delete the pod on success, why would I ever turn this off?"

Reply:
@ashb this has landed upstream but hasn't been put into a release yet, so I could change it now and it won't break anything.

Reply:
Agree, let's change this on the master to `delete_worker_pods_on_failure` - I think it makes more sense